### ISSUE TITLE
Not enough stock photos of hackers

### ISSUE THREAD

**Issue by youngJeezy858** [CyberWVU/cyberwvu-website#11]
You need to include some if you want to be taken seriously.  Here's a freebee - http://www.valentinatanni.com/public/2011/12/Hacker3cbcvbcv.jpg.jpg

**Comment by srshuff** [comment]
I second this

**Comment by bpsizemore** [comment]
I'll make a note to add some more good pics.

**Comment by camarox53** [comment]
Yeah this is a serious issue. It is affecting multiple users, need this fixed ASAP.
